src/Cake.Common.Tests/Unit/Tools/VSTest/VSTestRunnerTests.cs

```diff
@@ -204,7 +204,7 @@ public void Should_Use_SettingsFile_If_Provided()
             var result = fixture.Run();
 
             // Then
-            Assert.Equal("\"/Working/Test1.dll\" /Settings:Local.RunSettings", result.Args);
+            Assert.Equal("\"/Working/Test1.dll\" /Settings:\"/Working/Local.RunSettings\"", result.Args);
```

Comment:
@jnm2 since you noticed this issue when there was a path with a space in it, can you add another test that has a path with a space in it?